data.sql

```diff
@@ -23,6 +23,21 @@
 # Dump of table category
 # ------------------------------------------------------------
 
+DROP TABLE IF EXISTS `category`;
```

Comment:
Právě kvůli tomu slouží bin/console doctrine:schema:create a bin/console doctrine:schema:drop. To ti umožňuje vždy generovat správnou strukturu DB a tohle ti potom jen naplní daty tabulky.